client/webserver/site/src/js/order.ts

```diff
@@ -258,33 +263,86 @@ export default class OrderPage extends BasePage {
     const tmpl = Doc.parseTemplate(matchCard)
     tmpl.status.textContent = OrderUtil.matchStatusString(m)
 
-    const setCoin = (pendingName: string, linkName: string, coin: Coin) => {
-      const formatCoinID = (cid: string) => {
-        if (cid.startsWith(coinIDTakerFoundMakerRedemption)) {
-          const makerAddr = cid.substring(coinIDTakerFoundMakerRedemption.length)
-          return intl.prep(intl.ID_TAKER_FOUND_MAKER_REDEMPTION, { makerAddr: makerAddr })
-        }
-        return cid
-      }
```

Comment:
This is analogous to `setCoinHref` which could also be inlined (since it is used only once),

so since `setCoinHref` was moved outside seems natural to do same for `formatCoinID`, to simplify the `setMutableMatchCardElements` overall, or would you prefer to inline both ?